tools/local_server/LocalServer.java

```diff
@@ -72,12 +77,17 @@ private static void checkFlags() {
     }
   }
 
+  @Singleton
+  @Component(modules = { CommonModule.class })
+  public interface LocalServerComponent {
+    LocalServer getLocalServer();
+  }
+
   public static void main(String[] args) throws IOException, InterruptedException {
-    Flags.parse(args, LocalServer.class.getPackage(), CodeReviewService.class.getPackage(),
-        StorytellerService.class.getPackage());
+    Flags.parse(args, LocalServer.class.getPackage(), StorytellerService.class.getPackage());
```

Comment:
I think you need to add the `AuthService`'s package (`google.startupos.tools.localserver.service`).

Reply:
Yes, it fixed the problem.